setup.py

```diff
@@ -56,11 +56,10 @@
         'Operating System :: OS Independent',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: Only',
```

Comment:
Shouldn’t this be `Python :: 3 :: Only`?

Reply:
It should. Fixed it now, thanks!

https://pypi.org/pypi?%3Aaction=list_classifiers